tests/examples/nested-pyproject/dependencies.yaml

```diff
@@ -0,0 +1,24 @@
+files:
+  sparkly_unicorn:
```

Comment:
✨🦄😄

Reply:
ha this is one of my favorite things to use to make it absolutely clear that something is user-controlled arbitrary input and not something that has special meaning to the library

Reply:
This is making me think of PJReddie's website: https://pjreddie.com/